avro-converter/src/main/java/io/confluent/connect/avro/AvroData.java

```diff
@@ -85,12 +84,15 @@
 
   public static final String CONNECT_NAME_PROP = "connect.name";
   public static final String CONNECT_DOC_PROP = "connect.doc";
-  public static final String CONNECT_RECORD_DOC_PROP = "connect.record.doc";
-  public static final String CONNECT_ENUM_DOC_PROP = "connect.enum.doc";
   public static final String CONNECT_VERSION_PROP = "connect.version";
   public static final String CONNECT_DEFAULT_VALUE_PROP = "connect.default";
   public static final String CONNECT_PARAMETERS_PROP = "connect.parameters";
   public static final String CONNECT_INTERNAL_TYPE_NAME = "connect.internal.type";
+  public static final String AVRO_RECORD_DOC_PROP = NAMESPACE + ".record.doc";
+  public static final String AVRO_ENUM_DOC_PREFIX_PROP = NAMESPACE + ".enum.doc.";
+  public static final String AVRO_FIELD_DOC_PREFIX_PROP = NAMESPACE + ".field.doc.";
+  public static final String AVRO_FIELD_DEFAULT_FLAG_PROP = NAMESPACE + ".field.default";
```

Comment:
Let's comment this property, something like "This property is used to determine whether a default value in the Connect schema originated from an Avro field default."